Patch by Robert Scheck <robert@fedoraproject.org> for zarafa >= 7.0.0, which only
removes the paths from autoconf/automake related files, that don't exist in the OSS
version of Zarafa. Having these paths not removed causes autoreconf to fail, but a
re-run of autoreconf is needed in order to get the rpaths completely out of all the
libraries and binaries built during the make process.

--- zarafa-7.0.0/configure.ac			2011-01-20 21:30:52.000000000 +0100
+++ zarafa-7.0.0/configure.ac.package		2011-01-23 02:31:12.086844857 +0100
@@ -760,18 +760,6 @@
 		version
 		specialbuild
 ])
-dnl non-oss files
-if test -d `dirname $0`/licensed; then
-AC_CONFIG_FILES([
-		liblicense/Makefile
-		licensed/Makefile
-		ECtools/zarafa-backup/Makefile
-		ECtools/zarafa-backup/helpers/Makefile
-		ECtools/zarafa-report/Makefile
-		ECtools/libarchiver/Makefile
-		ECtools/zarafa-archiver/Makefile
-])
-fi
 
 AC_OUTPUT
 
--- zarafa-7.0.0/Makefile.am			2011-01-20 21:33:43.000000000 +0100
+++ zarafa-7.0.0/Makefile.am.package		2011-01-23 02:32:00.539820639 +0100
@@ -1,9 +1,5 @@
 ACLOCAL_AMFLAGS = -I autoconf
 
-if ! OSS_ONLY
-PROSUBS = liblicense licensed
-endif
-
 SUBDIRS = common libfreebusy zarafa-libsync mapi4linux $(PROSUBS) provider libicalmapi inetmapi php-ext spooler gateway caldav ECtools installer po doc
 
 if WITH_SWIG
@@ -25,10 +21,6 @@
 common/ecversion.h: versionheader.sh
 	sh ${top_srcdir}/versionheader.sh > common/ecversion.h
 
-if ! OSS_ONLY
-CLEANFILES=common/ecversion.h
-endif
-
 dist-hook:
 	find $(distdir) -type d -name .svn -exec rm -rf {} \; 2>/dev/null || true
 if OSS_ONLY
--- zarafa-7.0.0/ECtools/Makefile.am		2011-01-20 21:33:42.000000000 +0100
+++ zarafa-7.0.0/ECtools/Makefile.am.package	2011-01-23 02:32:10.426741530 +0100
@@ -1,7 +1,3 @@
-if ! OSS_ONLY
-PROSUBS = zarafa-backup zarafa-report libarchiver zarafa-archiver
-endif
-
 if WITH_TESTTOOLS
 TESTSUBS = ECTestTools
 endif
